openedx/core/djangoapps/organization_api/urls.py

```diff
@@ -0,0 +1,19 @@
+"""
+URLs for the organization app.
+"""
+from django.conf.urls import patterns, url
+
+from openedx.core.djangoapps.organization_api.api import views
+
+
+ORGANIZATION_KEY_PATTERN = r'(?P<organization_key>[^/]*)'
```

Comment:
@mattdrayer @ziafazal can you please suggest the expected constraint here?
In the `edx-organization` models this is the field which I believe is being used to query as an organization key:
```python
short_name = models.CharField(max_length=255, db_index=True)
```